Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/util/ContactDiaryExtensions.kt

```diff
@@ -40,3 +28,32 @@ fun String.formatContactDiaryNameField(maxLength: Int): String {
     }
     return newName.take(maxLength)
 }
+
+fun View.focusAndShowKeyboard() {
+    /**
+     * This is to be called when the window already has focus.
+     */
+    fun View.showTheKeyboardNow() {
```

Comment:
Why is this nested?

Reply:
member functions can be accessed without creating an object and for good overview